### PR TITLE
Add strong windows and a camera to the POS AI core.

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
More windows, a camera and a dream.

## Why It's Good For The Game
Makes the AI slightly less weak. Slightly.

## Changelog
:cl:
add: AI core strengthened on the Pillar of Spring
/:cl:

### CODE REVIEW

File: _maps/map_files/Pillar_of_Spring/TGS_Pillar_of_Spring.dmm [hunk 1]
```diff
@@ -684,6 +684,16 @@
...
 /turf/open/floor/mainship/mono,
 /area/mainship/command/cic)
+"ct" = (
+/obj/structure/window/reinforced/toughened{
+	desc = "Metal Laticed Glass, it looks remarkably thick, it is probably bullet proof, it appears to have a caustic barrier";
+	dir = 4;
+	max_integrity = 1500;
+	name = "Protective AI Glass";
+	resistance_flags = 10
+	},
```
Comment (SplinterGP) [inline _maps/map_files/Pillar_of_Spring/TGS_Pillar_of_Spring.dmm:694]: make a new type for those, less var edits the better

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Comment by Ieteihowmee** [comment]
Absolutely yes. I have been thinking of reworking AI core on PoS to make it less trash, this is a good start. I applaud you!

**Comment by lbnesquik** [comment]
It fixed now

File: code/datums/emergency_calls/sectoid.dm [hunk 2]
```diff
@@ -1,7 +1,7 @@
 //Sectoids
 /datum/emergency_call/sectoid
 	name = "Sectoid Expedition"
-	base_probability = 26
+	base_probability = 20
```
Comment (SplinterGP) [inline code/datums/emergency_calls/sectoid.dm:4]: oops ?

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Comment by lbnesquik** [comment]
@SplinterGP My master branch had the sectoid PR on it. I finally purged it and force pushed into this branch. 
It should work perfectly now.

File: code/game/objects/structures/window.dm [hunk 3]
```diff
@@ -310,6 +310,17 @@
...
 
+//For the sulaco and POS AI core.
+/obj/structure/window/reinforced/extratoughened
+	name = "protective AI glass"
+	desc = "Heavily reinforced glass with many layers of a rod matrice. This is rarely used for anything but the most important windows"
+	icon_state = "rwindow"
+	basestate = "rwindow"
+	max_integrity = 1500
+	reinf = TRUE
+	resistance_flags = 10 // I have no clue what those are.
```
Comment (SplinterGP) [inline code/game/objects/structures/window.dm:321]: unacidable and xeno_damageable i suppose ?

Comment (lbnesquik) [reply code/game/objects/structures/window.dm:321]: I do not know, that's what the hard coded stuff had. You would have to ask the other person what those do.

**Comment by lbnesquik** [comment]
aaaaaaaa now there is a merge conflict this is so sad

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**